certbot/tests/ocsp_test.py

```diff
@@ -72,21 +72,35 @@ def test_init(self, mock_exists, mock_popen, mock_log):
     @mock.patch('certbot.ocsp._determine_ocsp_server')
     @mock.patch('certbot.util.run_script')
     def test_ocsp_revoked(self, mock_run, mock_determine):
+        import pytz
```

Comment:
nit: I think we should just import this once at the top level of the file rather than in the various methods that need it.